computer-use-demo/computer_use_demo/loop.py

```diff
@@ -62,6 +62,15 @@ class APIProvider(StrEnum):
 </IMPORTANT>"""
 
 
+def debug_print(*args):
```

Comment:
We would prefer to use the python logging module; also be aware of `ANTHROPIC_LOG=debug` env var respected by the SDK.